CMakeLists.txt

```diff
@@ -97,32 +135,57 @@ else()
     message(ERROR "libpowerloader used as neither shared or static library: not supported")
 endif()
 
-# Custom build flags
-if(MSVC)
+target_include_directories(
+    libpowerloader
+    PUBLIC $<BUILD_INTERFACE:${POWERLOADER_INCLUDE_DIR}>
+    $<INSTALL_INTERFACE:include>
+)
+
+if (MSVC)
+    target_compile_definitions(libpowerloader PUBLIC -DNOMINMAX)
     target_compile_options(libpowerloader
         PUBLIC
```

Comment:
I'm thinking that having them as PUBLIC might be a mistake, the flags will be set for the users too...